package.json

```diff
@@ -20,5 +26,12 @@
     "lessc": "^1.0.2",
     "onchange": "^7.0.2",
     "prettier": "^2.1.1"
+  },
+  "dependencies": {
```

Comment:
Can you change these to "devdependencies" as they are required for development purposes and not for sButtons to work on its own?

Reply:
Ok